kinova_driver/src/kinova_joint_trajectory_controller.cpp

```diff
@@ -219,25 +236,22 @@ void JointTrajectoryController::pub_joint_vel(const ros::TimerEvent&)
 //                          std::endl <<" joint_velocity_msg.joint5: " << joint_velocity_msg.joint5 * M_PI/180 <<
 //                          std::endl <<" joint_velocity_msg.joint6: " << joint_velocity_msg.joint6 * M_PI/180 );
 
-        pub_joint_velocity_.publish(joint_velocity_msg);
+        //pub_joint_velocity_.publish(joint_velocity_msg);
```

Comment:
The publish is commented out. This controller doesn't do anything. 

I know this is rather confusing, but I was to concerned to rip the whole controller out, so I just prevented it from publishing joint velocity commands.